repos/system_upgrade/common/libraries/rhui.py

```diff
@@ -566,15 +580,29 @@ def mk_rhui_setup(clients=None, leapp_pkg='', mandatory_files=None, optional_fil
             ],
         },
     },
+    '9to10': {
+        'alibaba': {
+            'src_pkg': 'aliyun_rhui_rhel9',
+            'target_pkg': 'aliyun_rhui_rhel10',
+            'leapp_pkg': 'leapp-rhui-alibaba',
+            'leapp_pkg_repo': 'leapp-alibaba.repo',
+            'files_map': [
+                ('content.crt', RHUI_PKI_PRODUCT_DIR),
+                ('key.pem', RHUI_PKI_DIR),
+                ('leapp-alibaba.repo', YUM_REPOS_PATH)
+            ],
+        },
+    }
```

Comment:
This is no longer necessary - the dictionary should not be used anywhere if I am right. Nevertheless, your change is OK so no need to remove it.